osler/workup/models.py

```diff
@@ -90,9 +91,7 @@ def infer_coordinators(self):
         return coordinator_set
 
 
-class AttestableNote(Note):
-    class Meta:
-        abstract = True
+class AttestationMixin(models.Model):
```

Comment:
This is such a great solution, I literally thought we were already doing it this way!